### PR TITLE
[WIP]Introducing ssa for azure.

### PR DESCRIPTION
**Pull request by anurag03**
Purpose or Intent
=================
Introducing ssa for azure provider

{{pytest: -v -k 'test_ssa_vm' --long-running}}

### CODE REVIEW

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 1]
```diff
@@ -112,7 +113,15 @@ def vm_analysis_provisioning_data(provider, analysis_type):
     vma_data = provider.data.vm_analysis_new
     provisioning_data = vma_data.provisioning
 
-    if not isinstance(provider, CloudProvider):
+    if provider.type == 'azure':
+        provisioning_data.setdefault('instance_type', vma_data.provisioning.instance_type)
+        provisioning_data.setdefault('virtual_net',
```
Comment (lkhomenk) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: I'd recommend using `cloud_network` - it has all required values - split it. virtual_net is used in wrapanapi/REST

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
@anurag03 Please rebase or close this PR, depending on whether or not you are able to continue pursuing the change.

@sbulage Can you assist Anurag here, or take over the PR to push it to resolution?

Thanks!

**Comment by sbulage** [comment]
@mshriver I will take a look in it. Thanks :)

**Pull request closed without merging**